.env.SAMPLE

```diff
@@ -0,0 +1,8 @@
+# Copy this file to .env (which is in the .gitignore)
+# See README.md for instructions on obtaining values
+
+AUTH0_DOMAIN=copy-value-from-auth0
+AUTH0_CLIENT_ID=copy-value-from-auth0
+AUTH0_CLIENT_SECRET=copy-value-from-auth0
+
+MONGODB_URI=mongodb+srv://adminuser:ZUONH9nRtRniyBRC@cluster0-6c3fw.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
Do we want the live mongodb uri here?

Reply:
No, we do not.

Reply:
Though that is actually not a live address.